spec/index.bs

```diff
@@ -7625,6 +7657,13 @@ dictionary GPUSwapChainDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
+Issue: A {{GPUPredefinedColorSpace}} is necessary here.
+When we add float-format swap chains, we may need to choose between extended-srgb and
+clamped-srgb (if those are distinct upstream).
+Extended-srgb and clamped-srgb are the same for non-float-format swap chains, but for
+float-format swap chains, extended-srgb is probably the most reasonable behavior:
+float + clamped-srgb just gets you high-bit-depth, but not high-dynamic-range.
```

Comment:
I would also drop this as too speculative, or at least remove the actual speculation.
> Extended-srgb and clamped-srgb are the same for non-float-format swap chains, but for not for
float-format swap chains.